Makefile

```diff
@@ -13,6 +13,7 @@ BIN_NAME=$(IMAGE_NAME)_$(GOOS)_$(GOARCH)_$(VERSION)
 all: build
 
 build:
+	export GO111MODULE=on
```

Comment:
Is `export` really needed or could this be prepended to the `go build` command like the other envs?

Reply:
Removed, thanks!